TimberAPIExample/Examples/CustomObjectRegistryExample/CustomObjectAddExample.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Timberborn.AssetSystem;
+using Timberborn.FactionSystem;
+using Timberborn.FactionSystemGame;
+using Timberborn.SingletonSystem;
+using TimberbornAPI;
+using TimberbornAPI.AssetLoaderSystem.AssetSystem;
+using TimberbornAPI.LocalizationSystem;
+using UnityEngine;
+
+namespace TimberAPIExample.Examples.CustomObjectRegistryExample
+{
+    public class CustomObjectAddExample : IInitializableSingleton
+    {
+        private readonly IAssetLoader _assetLoader;
+        private readonly IResourceAssetLoader _resourceAssetLoader;
+
+        public CustomObjectAddExample(IAssetLoader assetLoader, IResourceAssetLoader resourceAssetLoader)
+        {
+            _assetLoader = assetLoader;
+            _resourceAssetLoader = resourceAssetLoader;
+        }
+
+        // Add a building to the custom object registry
+        public void Initialize()
+        {
+            // Load the Building with Prefab (use Thunderkit to make a bundle)
+            // AssetLoader is part of TimberAPI
+            GameObject customObject = _assetLoader.Load<GameObject>("TimberAPIExample/testprefab.bundle/testprefab");
+            Debug.Log("Loaded Building: " + customObject.name);
+            // Let TimberAPI load labels for you - just add to lang/<enUS>.txt
+            customObject.AddComponent<BasicLabeledPrefab>();
```

Comment:
This might be silly, but the BasicLabeledPrefab adds labels, but since inherits from LabelelPrefab it could be a good idea to show in the example how to add a Sprite to it?

Reply:
I have no idea how to do that - I wish I could